### ISSUE TITLE
100% cpu

### ISSUE THREAD

**Issue by simply-does-it** [moneymanagerex/moneymanagerex#1388]
Your OS:
- [x] Windows
- [ ] Mac OSX
- [ ] Linux 
- [ ] Android -> please report bugs to moneymanagerex/android-money-manager-ex repo!

MMEX version:
- [x] 1.4.x
- [x] 1.3.x
- [ ] 1.2.x
- [ ] 1.1 or older
- [ ] development version (git master branch)

**Comment by simply-does-it** [comment]
Windows 10 desktop
I have used MMEX with no problem for 17 years. Now it uses 90-100% cpu. (This is why this issue has had a bumpy start and I have not done much of what's required.
I'm using MMEX 1.3.3, it says it's up-to-date. On a suggestion from another forum, I installed  1.4 Alpha.
I have stripped the Db back so the 15 accounts only go back to January 2016.

I'm using MMEX 1.3.3, it says it's up-to-date. On a suggestion from another forum, I installed 1.4 Alpha, this has not helped. Nor has adding 127.0.0.1 as an exception to Windows firewall.
I am not a techie, but can generally follow instruction.

I would be grateful for advice on how to solve this.
